test/test_numba_integration.py

```diff
@@ -309,6 +309,30 @@ def test_from_cuda_array_interface(self):
                 torch_ary += 42
                 self.assertEqual(torch_ary.cpu().data.numpy(), numpy.asarray(numba_ary) + 42)
 
+    @unittest.skipIf(not TEST_NUMPY, "No numpy")
+    @unittest.skipIf(not TEST_CUDA, "No cuda")
+    @unittest.skipIf(not TEST_NUMBA_CUDA, "No numba.cuda")
+    def test_from_cuda_array_interface_inferred_strides(self):
+        """torch.as_tensor(numba_ary) should have correct inferred (contiguous) strides"""
```

Comment:
Nit: I think `numba_ary` read better as `numba_array`; I haven't seen too many usages of "ary" to mean array in the past

Reply:
I agree -- I was just following the style of the rest of the test here (which seems useful, since this test should just be combined with others once we make the test less strict).